changelog/unreleased/change-ocis-init.md

```diff
@@ -1,10 +1,12 @@
 Change: Introduce `ocis init` and remove all default secrets
 
-We've removed all default secrets. This means you can't start oCIS any longer
-without setting these via environment variable or configuration file.
+We've removed all default secrets and the hardcoded UUID of the user `adamin.
```

Comment:
This looks wrong. I guess you mean "`admin`" or "`adama`"

Is there a ` also missing?

Reply:
Fixed